client/packages/invoices/src/Returns/modals/InboundReturn/ItemSelector.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { FC } from 'react';
+import {
+  ModalRow,
+  ModalLabel,
+  Grid,
+  useTranslation,
+  Box,
+} from '@openmsupply-client/common';
+import { StockItemSearchInput } from '@openmsupply-client/system';
+import { useReturns } from '../../api';
+
+interface ItemSelectorProps {
+  itemId: string | undefined;
+  disabled: boolean;
+  onChangeItemId: (id: string) => void;
+}
+
+export const ItemSelector: FC<ItemSelectorProps> = ({
+  itemId,
+  disabled,
+  onChangeItemId,
+}) => {
+  const t = useTranslation();
+
+  const { items } = useReturns.lines.inboundReturnRows();
+
+  return (
+    <Box marginBottom="14px">
+      <ModalRow>
+        <ModalLabel
+          label={t('label.item', { count: 1 })}
+          justifyContent="flex-end"
+        />
+        <Grid item flex={1}>
+          <StockItemSearchInput
+            autoFocus={!itemId}
+            openOnFocus={!itemId}
+            disabled={disabled}
+            currentItemId={itemId}
+            onChange={newItem => newItem && onChangeItemId(newItem.id)}
+            extraFilter={
+              disabled
+                ? undefined
+                : item =>
+                    !items?.some(existingItem => existingItem.id === item.id)
+            }
+          />
```

Comment:
In an attempt to not use up even more vertical real estate, I didn't include the unit here, as we have in the shipment modals... but should I include it?

Reply:
You mean the number of available units? It's in the dropdown when selecting an item -- I wonder if we can just keep it in there once the item is selected?
<img width="967" alt="Screenshot 2024-03-18 at 4 33 59 PM" src="https://github.com/msupply-foundation/open-msupply/assets/5456533/b67322c1-d1cc-48ce-8399-66d9a47afa4c">
(i.e. make it show up in here, like the Batch is:)
<img width="976" alt="Screenshot 2024-03-18 at 4 34 34 PM" src="https://github.com/msupply-foundation/open-msupply/assets/5456533/7e8611ac-28db-48b7-9029-0bcce6a8d754">

Reply:
Oh sorry, that was a bit unclear, I meant like we have in shipments:

![Screenshot 2024-03-19 at 11 17 39 AM](https://github.com/msupply-foundation/open-msupply/assets/55115239/d66981e4-396a-41e5-8ac6-8c1de3f81d99)

Reply:
I think it's ok not to include, especially since pack variant thing would show units/pack size together